extensions/file-manager/src/utils.tsx

```diff
@@ -239,16 +240,27 @@ export function SymlinkItem(props: { fileData: FileDataType; refresh: () => void
       />
     );
   } else {
-    return (
+   return (
       <List.Item
         id={filePath}
         title={props.fileData.name}
         icon={{ fileIcon: filePath }}
         subtitle={preferences.showFilePermissions ? props.fileData.permissions : ""}
-        actions={
+        accessories={originalFileData === undefined 
+          ? [
+              {
+                icon: { source: "link-slash-solid.svg", tintColor: Color.Red },
+              },
+            ]
+          : []}
```

Comment:
```suggestion
        accessories={
          !originalFileData
            ? [{ icon: { source: Icon.ExclamationMark, tintColor: Color.Red }, tooltip: "Broken Symlink" }]
            : []
        }
```

Reply:
Raycast provides a good set of [icons](https://developers.raycast.com/api-reference/user-interface/icons-and-images). No need for external icons.

Reply:
i added tooltip, so users know what's up!

Reply:
That looks good but just out of curiosity is there a reason why we wouldn't want external icons? Just cause the broken link icon gives more information without requiring the user to hover.